packages/propel/src/dialog/root.tsx

```diff
@@ -34,6 +34,18 @@ export interface DialogTitleProps extends React.ComponentProps<typeof BaseDialog
   children: React.ReactNode;
 }
 
+export interface DialogPortalProps extends React.ComponentProps<typeof BaseDialog.Portal> {
```

Comment:
prefer types over interfaces for prop types, you don't need declaration merging for props